www/scripts/productlist/ListOfProducts.js

```diff
@@ -139,7 +139,9 @@ function (declare, domClass, domConstruct, ItemFileWriteStore, topic,
         { name : '&nbsp;', field : 'status', cellClasses : 'status', width : '20px', noresize : true },
         { name : '&nbsp;', field : 'icon', cellClasses : 'product-icon', width : '40px', noresize : true },
         { name : 'Name', field : 'name', cellClasses : 'product-name', width : '25%' },
-        { name : 'Description', field : 'description', styles : 'text-align: left;', width : '70%' }/*,
+        { name : 'Description', field : 'description', styles : 'text-align: left;', width : '70%' },
+        { name : 'Number of runs', field : 'runCount', styles : 'text-align: center;', width : '25%' },
+        { name : 'Date of latest run update', field : 'latestRunUpdateDate', styles : 'text-align: center;', width : '25%' }/*,
```

Comment:
Is it good that we say `update` here? Perhaps there could be a less potentially misleading (and shorter) column?

`Latest store to product`

The fact that it is a "date" is trivial from looking at the possible values.